test/tilesets_test.py

```diff
@@ -0,0 +1,19 @@
+import higlass.tilesets as hgti
+
+import pandas as pd
+
+def test_dfpoints():
+    df = pd.DataFrame({
+        'a': [1.0, 2.0, 3.0],
+        'b': [5.0, 6.0, 7.0],
+        'x': ['hi', 'there', 'you']
+    })
+
+    ts = hgti.dfpoints(
+        df, 'a', 'b', 'x'
```

Comment:
I'm confused. Does the `x` passed to `dfpoints` refer to a uuid or a column of `df`?